app/views/organizations/show.html.erb

```diff
@@ -1,109 +1,126 @@
 <h1>
   <%= link_to @organization, organization_path(@organization) %>
 </h1>
+
 <div class="row">
-  <div class="col-sm-4">
+  <div class="col-sm-6">
     <div class="panel panel-default">
       <div class="panel-heading">
         <h3 class="panel-title">
           <%= t '.contact_information' %>
         </h3>
       </div>
-      <div class="panel-body">
+      <div class="panel-body panel-organizer">
         <p>
```

Comment:
Semantically speaking, I would use an `<ul>` or even better a [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl), rather than a `<p>` with `<br/>`s.

Reply:
I agree, but @sseerrggii told me to make the minimun changes bacause the PR was already too big. The <p>'s and <br>'s were already in the code and I didn't change it. I should change it @nadalsol ?

Reply:
I think every thing in order to improve html & css validated by @nadalsol can be done in this PR :muscle:

Reply:
done